.circleci/config.yml

```diff
@@ -61,7 +74,7 @@ workflows:
                 - master
       - deploy-docs:
           requires:
-            - build-docs
+            - smoke-test
```

Comment:
I guess its safest to keep it dependent on the tests since theoretically you may have made some doc changes related to code changes that don't work and then the docs aren't really correct anymore. Not sure if thats really a case that would come up.

I'm fine changing it if you think its better to do that regardless of the smoke test result.

Reply:
Agree, we can keep it as it is to be safe here. 👍

Reply:
I'd opt to avoid any deploys unless the build passes the tests.